starter-code/css/main.css

```diff
@@ -6,3 +6,112 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font-family: Helvetica,Arial,sans-serif;
+  font-weight: lighter;
+  margin: 0;
+}
+
+/*HEADER SECTION*/
+
+header {
+  padding: 10px 40px 0px 40px;
+}
+
+header img {
+  width: 14%;
+}
+
+ul {
+  display: inline-block;
+  float: right;
+  list-style-type: none;
+}
+
+li {
+  display: inline-block;
+  padding-left: 50px;
+  font-size: 25px;
+}
+
+/*SECTION ONE*/
+
+.music {
+position: relative;
+}
+
+.texte {
+position: absolute;
+top: 35%;
+left: 50%;
+transform: translate(-50%, -50%);
+font-size: 18px;
+width: 1000px;
+}
+
+.music img {
+    width: 100%;
+    height: auto;
+}
+
+
+.music h1, h2 {
+  color: white;
+}
+
+.music h1 {
+   font-size: 80px;
+   text-align: center;
+}
+
+.music h2 {
+  margin-top: -20px;
+  font-weight: lighter;
+  text-align: center;
+  line-height: 1.5;
+  font-size: 30px;
+}
+
+/* SECTION 2*/
+.what {
+  text-align: center;
+}
+
+.title h1 {
+  font-size: 35px;
+  text-decoration: underline;
```

Comment:
Yes. However to match the screenshot perfectly a border would be more appropriate.